doc/monitoring/install.rst

```diff
@@ -0,0 +1,48 @@
+.. _install:
+
+Install the latest version of Metrics
+=====================================
+
+.. _install-the_usual_way:
+
+The usual way
+-------------
+
+Обычно все зависимости включают в файл ``*.rockspec`` приложения и уже
+из него устанавливают все зависимости.
+
+Это делается так:
+
+#.  Add the ``metrics`` package to the dependencies in the ``.rockspec`` file.
+    Make sure that you are using version **0.3.0** or higher.
```

Comment:
0.3.0 is a requirement for using a cartridge role. It doesn't make much sense here, in non-cartridge-related guide.